### PR TITLE
A small StFcsDb update

### PR DESCRIPTION
**Pull request by akioogawa**
Adding a function to get STAR coordinate XYZ for a StFcsCluster.

### CODE REVIEW

**Review by starsdong — APPROVED** [review]
Looks straightforward

**Comment by starsdong** [comment]
Daniel, could you take a quick review of this update? I think it still needs your approval as the code owner before merging.

File: StRoot/StFcsDbMaker/StFcsDb.h [hunk 1]
```diff
@@ -174,7 +174,8 @@ class StFcsDb : public TDataSet {
   //! get the STAR frame cooridnates from other way
   StThreeVectorD getStarXYZfromColumnRow(int det,float col, float row, float FcsZ=-1.0) const; //from column/row[cell size unit]
   StThreeVectorD getStarXYZ(int det, int col, int row, float FcsZ=-1.0) const;   //from column/row [cell unit]
-  StThreeVectorD getStarXYZ(StFcsHit* hit, float FcsZ=-1.0) const;                   //from StFcsHit
+  StThreeVectorD getStarXYZ(StFcsHit* hit, float FcsZ=-1.0) const;               //from StFcsHit
+  StThreeVectorD getStarXYZ(StFcsCluster* clu, float FcsZ=-1.0) const;           //from StFcsCluster
```
Comment (plexoos) [inline StRoot/StFcsDbMaker/StFcsDb.h]: I'd define the parameter as const

```suggestion
  StThreeVectorD getStarXYZ(const StFcsCluster* clu, float FcsZ=-1.0) const;           //from StFcsCluster
```

Comment (akioogawa) [reply StRoot/StFcsDbMaker/StFcsDb.h]: Done. For both with StFcsClutser and StFcsHit.

**Review by plexoos — APPROVED** [review]

**Comment by starsdong** [comment]
@jdbrice Daniel, any further comments before we merge?

**Comment by starsdong** [comment]
Hi Dmitri, could you please merge this PR? Thanks

**Pull request merged**